src/ExampleNets/regression/ConvertComplexToRealMatrix.srn5

```diff
@@ -0,0 +1,218 @@
+					<name>Output</name>
+					<id>0</id>
+				</port1_>
+				<moduleId2_>ConvertComplexToRealMatrix:0</moduleId2_>
+				<port2_>
+					<name>InputComplexMatrix</name>
+					<id>0</id>
+				</port2_>
+			</item>
+			<item>
+				<moduleId1_>CreateMatrix:0</moduleId1_>
+				<port1_>
+					<name>EnteredMatrix</name>
+					<id>0</id>
+				</port1_>
+				<moduleId2_>ConvertRealToComplexMatrix:0</moduleId2_>
+				<port2_>
+					<name>RealPartMatrix</name>
+					<id>0</id>
+				</port2_>
+			</item>
+			<item>
+				<moduleId1_>CreateMatrix:1</moduleId1_>
+				<port1_>
+					<name>EnteredMatrix</name>
+					<id>0</id>
+				</port1_>
+				<moduleId2_>ConvertRealToComplexMatrix:0</moduleId2_>
+				<port2_>
+					<name>ComplexPartMatrix</name>
+					<id>0</id>
+				</port2_>
+			</item>
+		</connections>
+	</networkInfo>
+	<modulePositions class_id="15" tracking_level="0" version="0">
+		<count>4</count>
+		<item_version>0</item_version>
+		<item class_id="16" tracking_level="0" version="0">
+			<first>ConvertComplexToRealMatrix:0</first>
+			<second class_id="17" tracking_level="0" version="0">
+				<first>-7.60000000000000000e+01</first>
+				<second>7.60000000000000000e+01</second>
+			</second>
+		</item>
+		<item>
+			<first>ConvertRealToComplexMatrix:0</first>
+			<second>
+				<first>-7.60000000000000000e+01</first>
+				<second>-7.60000000000000000e+01</second>
+			</second>
+		</item>
+		<item>
+			<first>CreateMatrix:0</first>
+			<second>
+				<first>-1.82000000000000000e+02</first>
+				<second>-2.20000000000000000e+02</second>
+			</second>
+		</item>
+		<item>
+			<first>CreateMatrix:1</first>
+			<second>
+				<first>7.30000000000000000e+01</first>
+				<second>-2.25000000000000000e+02</second>
+			</second>
+		</item>
+	</modulePositions>
+	<moduleNotes class_id="18" tracking_level="0" version="0">
+		<count>0</count>
+		<item_version>0</item_version>
+	</moduleNotes>
+	<connectionNotes>
+		<count>0</count>
+		<item_version>0</item_version>
+	</connectionNotes>
+	<moduleTags class_id="19" tracking_level="0" version="0">
+		<count>4</count>
+		<item_version>0</item_version>
+		<item class_id="20" tracking_level="0" version="0">
+			<first>ConvertComplexToRealMatrix:0</first>
+			<second>-1</second>
+		</item>
+		<item>
+			<first>ConvertRealToComplexMatrix:0</first>
+			<second>-1</second>
+		</item>
+		<item>
+			<first>CreateMatrix:0</first>
+			<second>-1</second>
+		</item>
+		<item>
+			<first>CreateMatrix:1</first>
+			<second>-1</second>
+		</item>
+	</moduleTags>
+	<disabledModules class_id="21" tracking_level="0" version="0">
+		<count>0</count>
+		<item_version>0</item_version>
+	</disabledModules>
+	<disabledConnections>
+		<count>0</count>
+		<item_version>0</item_version>
+	</disabledConnections>
+	<moduleTagLabels class_id="22" tracking_level="0" version="0">
+		<count>0</count>
+		<item_version>0</item_version>
+	</moduleTagLabels>
+	<loadTagGroups>0</loadTagGroups>
+</networkFile>
+</boost_serialization>
+
```

Comment:
No errors here, but the test network should have something connected to the module you're testing, at least to assert it creates some output. In this case, ReportComplexMatrixInfo would suffice.